src/tr2/decomp/savegame.c

```diff
@@ -942,8 +1048,8 @@ int32_t S_SaveGame(
     return true;
 }
 
-int32_t S_LoadGame(
-    void *const save_data, const size_t save_size, const int32_t slot_num)
+#include <libtrx/log.h>
```

Comment:
Do you want to retain the logging here for now?

Reply:
Oh no, I'll get rid of it right away.

Reply:
Done